src/main/webapp/main.js

```diff
@@ -30,3 +32,47 @@ Promise.all([mapPromise, domPromise]).then(() => {
 function initMap() {
   myMap = new ChapMap();
 }
+
+
+connection.onopen = () => {
+};
+
+connection.onclose = () => {
+};
+
+connection.onerror = (error) => {
+  console.error(error);
+};
+
+connection.onmessage = (event) => {
+  let li = document.createElement('li');
+  
+  var obj = JSON.parse(event.data); 
+
+  if(obj.type !== 'MSG_RECV'){
+      // let Alice deal with it;
```

Comment:
The convention that Google uses for TODO comments is the following:
`// TODO(<ldap>): <message>`

Please update to something like
`// TODO(alicevlasov): Handle MSG_RECV`